app/models/trainee.rb

```diff
@@ -1,2 +1,19 @@
+class LockedAttributeError < StandardError; end
```

Comment:
should this be on it lonesome?

Reply:
I've moved it into `app/models/locked_attribute_error.rb`. I don't mind it being in the model as it currently only gets used there. Another approach might be to have an `app/models/errors.rb` file and put it in there. That would give us a place to put others as they're generally only one liners. What do you think?